javascript/powercommand_using_SDKv3/task_provider.js

```diff
@@ -0,0 +1,22 @@
+const getTaskRefs = async (tasksApiInstance, projectId) => {
+  console.log('Retrieving all task references for project', projectId);
+  let opts = {
+    opt_fields: 'name,assignee,completed,resource_type,resource_subtype'
```

Comment:
The `assignee` filter and `notes` filters won't work unless you ask for those properties in `opt_fields`

FROM:
```
    opt_fields: 'name,assignee,completed,resource_type,resource_subtype'
```

TO:
```
    opt_fields: 'name,assignee.name,completed,resource_type,resource_subtype,notes'
```

Reply:
updated the opt_fields to:  'name,assignee.name,completed,resource_type,resource_subtype,notes'